inc/saiacl.h

```diff
@@ -380,6 +403,65 @@ typedef enum _sai_acl_table_group_attr_t
 
 } sai_acl_table_group_attr_t;
 
+/**
+ * @brief Attribute Id for acl_table_chain_group
+ */
+typedef enum _sai_acl_table_chain_group_attr_t
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_ACL_TABLE_CHAIN_GROUP_ATTR_START,
+
+    /**
+     * @brief ACL table sub group type
+     *
+     * ACL table group type represents the way various ACL tables within this
+     * ACL table group perform their lookups. There are two optional values:
+     * Sequential - All the ACL tables are looked up in a sequential order,
+     * which is based on the ACL table priorities and only one ACL entry is matched
+     * with its corresponding ACL entry action applied. In case two ACL tables
+     * have the same priority they are looked up on a first come basis.
+     * Parallel - All the ACL tables within the ACL table groups are looked up
+     * in parallel and non-conflicting actions are resolved and applied from
+     * multiple matched ACL entries (each from different ACL tables of this group).
+     * Conflicting actions are resolved based on the ACL table priorities.
+     *
+     * @type sai_acl_table_chain_group_type_t
+     * @flags CREATE_ONLY
+     * @default SAI_ACL_TABLE_CHAIN_GROUP_TYPE_SEQUENTIAL
+     */
+    SAI_ACL_TABLE_CHAIN_GROUP_ATTR_TYPE = SAI_ACL_TABLE_CHAIN_GROUP_ATTR_START,
+
+    /**
+     * @brief ACL table sub group stage
+     *
+     * ACL table sub group stage represents the cascading stage in the pipeline.
+     * Lower numbered stage comes before the higher numbered stage.
+     *
+     * @type sai_acl_table_chain_group_stage_t
```

Comment:
Please consider having this as uint32\_t with default value as 0. This will help keep it extensible without having to add new enums.